src/Akka.Remote.Hosting/Akka.Remote.Hosting.csproj

```diff
@@ -5,10 +5,6 @@
     <Description>Akka.Remote Microsoft.Extensions.Hosting support.</Description>
   </PropertyGroup>
 
-  <ItemGroup>
```

Comment:
Do you want to change this to reference the `README.md` in this directory instead?